packages/design-system-docs/src/pages/components/DateField/datefield.example.html

```diff
@@ -97,7 +97,7 @@
         <div class="ds-l-col--auto">
           <label class="ds-c-label ds-c-datefield__label" for="year"><span>Year</span></label>
           <input
-            class="ds-c-field ds-c-field--error ds-c-field--year"
+            class="ds-c-field ds-c-field--error ds-c-field--inverse ds-c-field--year"
```

Comment:
Is this still necessary?

Reply:
This is for this change (https://github.com/CMSgov/design-system/pull/847) - i think we missed updating this html example